vcd/datasource_vcd_storage_profile.go

```diff
@@ -32,6 +32,70 @@ func datasourceVcdStorageProfile() *schema.Resource {
 				Required:    true,
 				Description: "Name of storage profile",
 			},
+			"limit": &schema.Schema{
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: "Maximum number of storage bytes (scaled by Units) allocated for this profile. A value of 0 is understood to mean `maximum possible`",
+			},
+			"used_storage": &schema.Schema{
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: "Storage used, in Megabytes, by the storage profile",
+			},
+			"default": &schema.Schema{
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "True if this is default storage profile for this vDC. The default storage profile is used when an object that can specify a storage profile is created with no storage profile specified",
+			},
+			"enabled": &schema.Schema{
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "True if this storage profile is enabled for use in the vDC",
```

Comment:
Let's use `VDC` in all the messages of the PR.

Reply:
fixed